Python/xLinkingBookGUIPopup.py

```diff
@@ -293,7 +293,7 @@ def OnNotify(self,state,id,events):
                             PtSetOfferBookMode(self.key, self.IGetAgeFilename(), self.IGetAgeInstanceName())
                             PtSetShareSpawnPoint(self.IGetAgeSpawnPoint())
                             if self.IGetAgeFilename() == "PelletBahroCave":
-                                print "using Adam's plasma-magical 'PtSetShareAgeInstanceGuid' with pelletcaveguid = ",pelletCaveGUID
+                                PtDebugPrint("using Adam's plasma-magical 'PtSetShareAgeInstanceGuid' with pelletcaveguid = ",pelletCaveGUID)
```

Comment:
Disclaimer: this is not talking about me. I would have never written this shameless evil.

Reply:
But it's plasma-magical! Must be good.